### ISSUE TITLE
[DoD] Viewdemo menu closes when pressing ESC

### ISSUE THREAD

**Issue by 1337Smithy** [ValveSoftware/halflife#3713]
When using viewdemo console command, the menu used to control playback closes if you press ESC. This wasn't the case before the anniversary update, so now when you press escape to leave the main menu and watch the demo currently playing, you cannot control playback when going back to the main menu. Running viewdemo command again starts the demo from the start.

**Comment by dtugend** [comment]
Pretty annoying for people that want to convert demos into movies, had to recommend them to roll back to the "steam_legacy - Pre-25th Anniversary Build" instead, so please don't delete that build too soon.

**Comment by 1337Smithy** [comment]
> Pretty annoying for people that want to convert demos into movies, had to recommend them to roll back to the "steam_legacy - Pre-25th Anniversary Build" instead, so please don't delete that build too soon.

You can also spam console key ( ` ) to open / close the main menu. Annoying to do, and it means you need to close/minimise the console every time you open the menu again, but it works.

**Comment by exegg** [comment]
This is so obnoxious. How many ways could someone find to break a 20 year old game? The only true fix is use the steam_legacy branch

Video example of the issue.

https://github.com/user-attachments/assets/d9868160-e7c9-4854-8c7e-de9c84880b56

**Comment by sabianroberts** [comment]
Yeah it's annoying, though as a temporary fix you can bind a key to the command
